v0/destinations/customerio/transform.js

```diff
@@ -73,7 +73,7 @@ function responseBuilder(message, evType, evName, destination, messageType) {
     adduserIdFromExternalId(message);
   }
 
-  const userId = getFieldValueFromMessage(message, "userId");
```

Comment:
Is this working for identify calls with `anonymousId` too?

Reply:
Yes, it's functionality is intact